src/main/kotlin/org/sert2521/bunnybots2019/beddumper/BedDumperCommands.kt

```diff
@@ -0,0 +1,37 @@
+package org.sert2521.bunnybots2019.beddumper
+
+import kotlinx.coroutines.launch
+import org.sert2521.sertain.coroutines.periodic
+import org.sert2521.sertain.events.onTick
+import org.sert2521.sertain.subsystems.doTask
+import org.sert2521.sertain.subsystems.use
+
+suspend fun beddumper() = doTask {
```

Comment:
This suspend function is something we call a _command_. When this command (function) is called when a button on the joystick is pressed, it should do something. In this case, it will spin the bed dumper motor every 20ms.

Stylistically it's usually helpful to use verbs when naming command functions so they're really recognizable.

```suggestion
suspend fun dumpBed() = doTask {
```